spec/linked_list_spec.cr

```diff
@@ -3,13 +3,15 @@ require "./spec_helper"
 describe LinkedList do
   describe "#initialize" do
     it "initializes the list" do
-      LinkedList(Nil).new
+      list = LinkedList(Nil).new
+      list.size.should eq 0
     end
 
     it "initializes the list with the given values" do
-      list = LinkedList(Nil | Int32 | String).new(nil, 13, "foo")
+      list = LinkedList(Nil | Int32 | String).new(1, 13, "foo")
+      list.size.should eq 3
```

Comment:
I generally try to keep tests structured so they follow this pattern:

1. setup (optional)
2. invocation (the thing under test) 
3. assertions

Adding the assertion on size here makes the test also depend on the list's`size` function, which I'd like to avoid if possible.

Reply:
You're right, I can rework it.